src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -18,9 +18,18 @@ import {
 } from '../../data/constants';
 import { endDateHelp, enrollmentHelp, startDateHelp, titleHelp, pacingHelp } from '../../helpText';
 import DateTimeField from '../DateTimeField';
-import { getDateString } from '../../utils';
+import { getDateString, localTimeZone } from '../../utils';
 
 class BaseCreateCourseForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { organizations: [{ label: 'Select organization', value: '' }] };
```

Comment:
Do you need to use the state to handle the organization list? I think the original `processOrganizations()` method handled it as expected